group_consumer_test.go

```diff
@@ -46,7 +46,7 @@ func (sc *simpleSyncMock) XAck(ctx context.Context, stream, group string, ids ..
 
 func TestGroupConsumer_SimpleSync(t *testing.T) {
 	rdb := simpleSyncMock{}
-	cs := NewGroupConsumer[City](context.TODO(), &rdb, "g1", "c1", "s1", ">")
+	cs := NewGroupMultiStreamConsumer[City](context.TODO(), &rdb, "g1", "c1", map[string]string{"s1": ">"})
```

Comment:
The whole reason we added this feature was to make it usable with multiple streams 😅 

Let's keep the old `NewGroupConsumer` for existing tests with one channel. What I'd really like to see would be a test that uses multiple streams (and thus has a reason to actually use the `NewGroupMultiStreamConsumer` function)